src/components/NewRuntimeModal.js

```diff
@@ -83,6 +85,7 @@ const DiskSelector = ({ value, onChange }) => {
     id => h(Fragment, [
       label({ htmlFor: id, style: styles.label }, ['Disk size (GB)']),
       h(NumberInput, {
+        'aria-label': 'Disk size',
```

Comment:
should this be consistent with the above labels, i.e. "the disk size of the cloud environment"?